updates/99-fas.update

```diff
@@ -91,3 +91,10 @@ default:objectClass: groupOfPrincipals
 default:objectClass: ipaKrb5DelegationACL
 default:cn: fasjson-http-delegation
 default:ipaAllowedTarget: cn=ipa-ldap-delegation-targets,cn=s4u2proxy,cn=etc,$SUFFIX
+
+# create the container for the user agreements
+dn: cn=agreements,$SUFFIX
+default: objectclass: top
+default: objectclass: nsContainer
+default: cn: agreements
+plugin: update_managed_permissions
```

Comment:
Add this privilege before the plugin call and replace ``CA Administrator`` with ``Agreement Administrators`` (IPA uses plural form for most other privileges).

```
# Agreement Administrators
dn: cn=Agreement Administrators,cn=privileges,cn=pbac,$SUFFIX
default:objectClass: nestedgroup
default:objectClass: groupofnames
default:objectClass: top
default:cn: Agreement Administrators
default:description: Manage FAS user agreements
```